PocketSwift/PocketSwift/Packages/Core/Model/RelayData.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RelayData.swift
+//  PocketSwift
+//
+//  Created by Wilson Garcia on 3/23/19.
+//  Copyright © 2019 Wilson Garcia. All rights reserved.
+//
+
+import Foundation
+
+struct RelayData: Input {
+    let jsonrpc: String
+    let method: String
+    let params: [String]
```

Comment:
add another property "id", can be any number, in javascript I take actual time number representation, it should work without it but in case something happens in the future let's add it